src/canvas.ts

```diff
@@ -168,10 +168,12 @@ export interface CanvasAssignment {
 export class CanvasAssignmentWrapper {
     canvas_assignment: CanvasAssignment;
     canvas_general_infos: CanvasAssignmentEvent;
+    canvas_course_name: string;
 
-    constructor(canvas_assignment: CanvasAssignmentEvent) {
+    constructor(canvas_course:string, canvas_assignment: CanvasAssignmentEvent) {
```

Comment:
nit: Add consistent spacing between the parameter and its type. Also see line 262